### PR TITLE
Add `-DENABLE_WERROR=0` to `cmake` arguments to fix build

### PR DESCRIPTION
**Pull request by torokati44**
Based on: https://github.com/emscripten-core/emsdk/pull/1028

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by torokati44** [comment]
@conda-forge-admin, please rerender

**Comment by jakirkham** [comment]
Thanks @torokati44! 🙏 

Should we grab the version update from PR ( https://github.com/conda-forge/binaryen-feedstock/pull/56 ) as well?

**Comment by torokati44** [comment]
Yes, I made this PR specifically to allow the update to v110. Thanks! :)

**Comment by torokati44** [comment]
Hmm, I force-pushed the source branch, but it seems the CI runs on Azure are still running for a295ce00818ad5c2c4cba39d06f8083f4bf3964f, which is no longer on this branch...?
Am I seeing that right? Is this a problem? :D  Should I not have force-pushed? :no_mouth:

**Comment by torokati44** [comment]
I can also just open a "fresh" PR without the force-push if you would prefer that. :| Sorry about the mess...

**Comment by jakirkham** [comment]
Could you please rebase this change on PR ( https://github.com/conda-forge/binaryen-feedstock/pull/56 )?

**Comment by torokati44** [comment]
Yep, done!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-  sha256: 7fea5013da7f98daeeb6a0a60333e8aa917bcb0b1b418ba8531fe710d09041f2
+  sha256: 0f80250a02b94dd81bdedeae029eb805abf607fcdadcfee5ca8b5e6b77035601
   patches:
     # this fixes an issue where "some toolchains it is necessary to #define __STDC_FORMAT_MACROS before using macros like PRIxPTR"
     # cf: https://github.com/tensorflow/tensorflow/issues/12998#issuecomment-329179887
     - gcc_stdmacro.patch  # [linux]
 
 build:
-  number: 0
+  number: 1
```
Comment (jakirkham) [inline recipe/meta.yaml]: Since the version number is bumped, the build number can be set to `0`

```suggestion
  number: 0
```

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Thanks @torokati44! 🙏

**Comment by torokati44** [comment]
Thank you too @jakirkham!

**Pull request merged**